_posts/2023-06-11-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,195 @@
+
+Apx v2 still offers this possibility, but it does so through stacks that are configurable environments based on the user's needs.
+
+A stack is a set of instructions that can configure an environment based on any Linux distribution, adding packages and settings at the user's discretion. At any time, the user can create a new stack by defining a base Linux distribution, such as Arch Linux, and an optional list of packages. With this stack, the user can start one or more new subsystems based on it.
+
+Once a subsystem gets created, the user can use package manager commands (install, remove, update, etc.) or enter the subsystem, just like in Apx v1.
+
+Stacks are shareable and restoreable, so you can easily share them with your team. Apx v2 already has a set of preconfigured stacks to access all the major Linux distributions.
+
+Thanks to this new structure, the user can create stacks for each use case, for example, one with all the tools and libraries useful for development in various domains like 3D applications, music production, programming, etc.
+
+If something goes wrong, the subsystem can be restored by reinitializing it, resetting the environment to the state defined by the stack.
+
+### VSO, as a system package manager
+
+VSO v2 plays a crucial role in Vanilla OS Orchid. On top of managing update configurations, it will also handle the installation of Deb, Nix, and Apk packages through Waydroid.
+
+#### Vanilla OS Subsystem
+
+Since Apx v2 is a tool designed for developers and more advanced users, which will receive several other features for this target audience over time, we have decided to create a new subsystem specifically for the end user, who is typically less technologically inclined. In VSO v2, there is a subsystem based on Vanilla OS Pico. This subsystem will be a system-integrated container which supports systemd, allowing straightforward installation, removal, and management of packages, both graphically and through the command line.
+
+At any time, the user can access the command line to install packages using this subsystem or use the new graphical sideload utility to install Deb packages or Android APKs.
+
+![Installing an Apk package of Chromium using the sideload utility](/assets/uploads/sideload-utility-install.webp)
+
+![Uninstalling an Apk package of Chromium using the sideload utility](/assets/uploads/sideload-utility-uninstall.webp)
+
+We are working on making the subsystem easier to access by integrating it into the GNOME software. We can't promise this at this time. But it is in our plans.
+
+#### Waydroid
+
+Support for Android applications has been one of the most widely requested features by our community since the beginning of the project. Finally, this will be possible with VSO v2. The integration is similar to the Vanilla OS subsystem, as it will allow us to install Android APKs from the command line or through the Sideload application by simply downloading and opening an APK.
```

Comment:
Maybe mention that it is using waydroid behind the scene. "It is similar to the Vanilla OS subsystem" is a bit confusing.

Reply:
The title is literally Waydroid, and the similarity is explained in the paragraph.

Reply:
I mean, the title is Waydroid. The similarities we are referring to are user facing.